samples/msal-angular-v2-samples/angular10-sample-app/package.json

```diff
@@ -7,7 +7,7 @@
     "build": "ng build",
     "test": "ng test",
     "lint": "ng lint",
-    "e2e": "ng e2e"
+    "e2e": "jest"
```

Comment:
Can we also delete the existing `e2e` folder in each sample?

Reply:
Yes! Done.